src/alitra/alignment.py

```diff
@@ -0,0 +1,168 @@
+from typing import Literal, Tuple
+
+import numpy as np
+from numpy.linalg import norm  # type: ignore
+from scipy.spatial.transform import Rotation
+
+from alitra.transformations import transform_position
+
+from .models.map import Map
+from .models.position import Positions
+from .models.transform import Transform
+from .models.translation import Translation
+
+
+def align_maps(
+    map_from: Map,
+    map_to: Map,
+    rot_axes: Literal["x", "y", "z", "xyz"],
+    rsmd_treshold=0.4,
+) -> Transform:
+    return align_positions(
+        map_from.reference_positions,
+        map_to.reference_positions,
+        rot_axes,
+        rsmd_treshold,
+    )
```

Comment:
Do we need this wrapper function? Why can the user not use 'align_positions'? Or remove this function and rename 'align_positions' to 'align_maps'

Reply:
I made this for convenience, as my hope is that ISAR will only deal with maps, rather than a list of positions. What do you think?

Reply:
Then it makes sense to keep it like this. Resolving comment.